src/Moryx.ControlSystem/VisualInstructions/IInstructionResults.cs

```diff
@@ -34,4 +34,20 @@ public interface IInstructionInputResults : IInstructionResults
         /// </summary>
         void Invoke(string result, object input);
     }
+
+    /// <summary>
+    /// Extended interface with improved result values
+    /// </summary>
+    public interface IInstructionKeyResults : IInstructionResults
+    {
+        /// <summary>
+        /// Possibile results for the instructions
+        /// </summary>
+        InstructionResult[] PossibleResults { get; }
+
+        /// <summary>
+        /// Extended <see cref="IInstructionResults.Invoke(string)"/> with filled input object
```

Comment:
I'm not sure whats correct here, but the comment is coppied from the input results . The interface inherits from the IInstruction result however 🤔

Reply:
thx, extended the comment to indicate the changed result object and purpose